File: src/test/com/natpryce/snodge/InANutshell.java
package com.natpryce.snodge;

public class InANutshell {
    public static void main(String[] args) {
        JsonMutator mutator = new JsonMutator();
        String originalJson = "{\"x\": \"hello\", \"y\": [1,2,3]}";

        mutator.forStrings().mutate(originalJson, 10)
                .forEach(System.out::println);
    }
}

File: src/test/com/natpryce/snodge/InANutshell.kt
package com.natpryce.snodge

import java.util.Random

object InANutshell {
    @JvmStatic fun main(args: Array<String>) {
        val random = Random()
        val originalJson = "{\"x\": \"hello\", \"y\": [1,2,3]}"
        
        random.mutants(originalJson, 10, allMutagens().forStrings())
            .forEach(::println)
    }
}
